### PR TITLE
Add Deno guide

### PR DESCRIPTION
**Pull request by Return180bpm**
Added a guide for setting up excalibur in a Deno environment, as dicussed in [#2062](https://github.com/excaliburjs/Excalibur/issues/2062).

### CODE REVIEW

File: docs/02.1-bundlers.mdx [hunk 1]
```diff
@@ -130,3 +130,174 @@ Configuring webpack is not for the faint of heart, we recommend thoroughly readi
 [Our template](https://github.com/excaliburjs/template-ts-webpack) is a good starting point.
 
 </Note>
+
+### Deno
+
+[Deno](https://deno.land/) is a runtime for JavaScript and TypeScript and much more than a bundler, but we put this section here since you'll be doing a lot of bundling :).
+
+For Excalibur to work in a Deno envirionment, use a Content Delivery Network for making Node.js packages compatible with Deno: two popular ones are [esm.sh](https://esm.sh/) and [skypack.dev](https://www.skypack.dev/).
```
Comment (eonarheim) [inline docs/02.1-bundlers.mdx:138]:
```suggestion
For Excalibur to work in a Deno environment, use a Content Delivery Network for making Node.js packages compatible with Deno: two popular ones are [esm.sh](https://esm.sh/) and [skypack.dev](https://www.skypack.dev/).
```

File: docs/02.1-bundlers.mdx [hunk 2]
```diff
@@ -130,3 +130,174 @@ Configuring webpack is not for the faint of heart, we recommend thoroughly readi
...
+
+To keep it simple, we use esm.sh in the following examples, but feel free to make your own choice - at the time of writing the procedure for esm.sh and skypack.dev is nearly identical (for a difference, see the section [Types and IntelliSense](#types-and-intellisense)).
```
Comment (eonarheim) [inline docs/02.1-bundlers.mdx:140]:
```suggestion
To keep it simple, we use `esm.sh` in the following examples, but feel free to make your own choice - at the time of writing the procedure for `esm.sh` and `skypack.dev` is nearly identical (for a difference, see the section [Types and IntelliSense](#types-and-intellisense)).
```

File: docs/02.1-bundlers.mdx [hunk 3]
```diff
@@ -130,3 +130,174 @@ Configuring webpack is not for the faint of heart, we recommend thoroughly readi
...
+
+Learn more about the `type="module"` attribute [on MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules).
+
+</Note>
+
+#### Importing into a JavaScript or TypeScript File
+
+<Note>
+
+We have tested the following examples with version 0.26.0-alpha.264 of the excalibur package. We recommend you run at least that version for this to work. You can get [the latest excalibur version on npm](https://www.npmjs.com/package/excalibur).
```
Comment (eonarheim) [inline docs/02.1-bundlers.mdx:183]: We'll cut a point release soon, I'll make a note so I don't forget to update this :+1:

File: docs/02.1-bundlers.mdx [hunk 4]
```diff
@@ -130,3 +130,174 @@ Configuring webpack is not for the faint of heart, we recommend thoroughly readi
...
+
+Make sure to read the instructions of the extension. you might also need to install the Deno CLI and later initialize the extension in your IDE workspace.
+
+<Note variant="warning">
+
+It might also take a few seconds until the IntelliSense starts working. Deno is resolving the import URLs and downloading the necessary files in the background.
+
+</Note>
+
+**If you're using skypack.dev**, you might also need to add the `?dts` query parameter to the package name. your import should read something like this:
```
Comment (eonarheim) [inline docs/02.1-bundlers.mdx:287]: 👍

**Review by eonarheim — CHANGES_REQUESTED** [review]
@Return180bpm Thank you so much for this Deno addition!

* Couple small text tweaks suggested
* Remove the `package-lock.json` changes there aren't any `package.json` updates

**Comment by Return180bpm** [comment]
Hey @eonarheim, thank You for the speedy review and your corrections! They've been applied.

I also added a pretty big commit with changes to /docs/installation. I should have put this in here to begin with, but I forgot. It basically adds a section about CDNs and links to the newly written guide. (I also reformatted the intro to make it more readable.)

File: docs/01-installation.mdx [hunk 5]
```diff
@@ -5,7 +5,12 @@ path: /docs/installation
 
 ### Getting Excalibur
 
-There are several ways you can get Excalibur: as standalone packages from `npm` or `nuget`, as raw script files you can download, or by using one of our pre-made [template repos](#github-templates).
+There are several ways you can get Excalibur:
```
Comment (eonarheim) [inline docs/01-installation.mdx:8]: 👍

**Review by eonarheim — APPROVED** [review]
@Return180bpm Thanks! This is great!

**Pull request merged**